social-network/src/articles-yours.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+
+export default class YourArticle extends Component {
+    /* I recieve the following warning on this .js file as well as articles-missed.js and articles-view.js:
+    *
+    * 'Useless constructor  no-useless-constructor'
+    * 
+    * Am i not utilizing my constructor functon correctly?
+    */
+
+
+    constructor(props) {
+        super(props);
+    }
+
+    openArticle = (event) => {
+        window.open(this.props.article.link, '_blank');
+    }
+
+    render () {
+
+        // postedDate date format '2018-03-30'
+        let postedDate = '';
+        let jsonDate = this.props.article.postedDate;
+        let newPostedDate = jsonDate.slice(5,10);
+        let monthPosted = newPostedDate.slice(0,2);
+        let datePosted = newPostedDate.slice(3,6);
+
+        switch (monthPosted) {

```

Comment:
Creating an array of strings might be more efficient and readable. I've seen a lot of people use a package called Moment for date formatting. Also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString